src/lib/user/user-actor.ts

```diff
@@ -124,11 +127,25 @@ export class UserActor {
   }
 
   // TODO: Test UserActor::getDogReports
-  async getDogReports(args: { dogId: string }) {
+  async getDogReports(args: {
+    dogId: string;
+  }): Promise<
+    Result<BarkReport[], typeof CODE.FAILED | typeof CODE.ERROR_DOG_NOT_FOUND>
+  > {
```

Comment:
Remove the return type once we have migrated to DogProfileService::getDogReports

Reply:
Removed in 03baf59f